airbyte-config/config-models/src/main/java/io/airbyte/config/EnvConfigs.java

```diff
@@ -795,6 +795,12 @@ public String getDDDogStatsDPort() {
     return getEnvOrDefault(DD_DOGSTATSD_PORT, "");
   }
 
+  @Override
+  public List<String> getDDConstantTags() {
```

Comment:
Can we drop a test case in EnvConfigsTest to double check this parsing is correct?